HACKING.md

```diff
@@ -69,38 +69,17 @@ Guides for making changes to the ubuntu.com codebase.
 
 The basic navigation structure of the site is listed in `navigation.yaml`. This file lists all the top-level section pages and their children and grandchildren. The top menu, footer menu and breadcrumb navigation are all built from this list.
 
-The file should be of the following format:
-
-```yaml
-{ section-identifier }:
-  title: { Section title }
-  url_path: /{section-url}
-
-  children:
-    - title: { Child title }
-      url_path: /{child-url}
-
-      children:
-        - title: { Grandchild }
-          url_path: /{grandchild-url}
-
-    - title: { Hidden child }
-      url_path: { child-url }
-      hidden: True
-```
```

Comment:
I feel this section should still be left as it gives a visual cue of the structure of the file

Reply:
I deleted this as you can find an example at the top of [navigation.yaml](https://github.com/canonical/ubuntu.com/blob/wd-23038/navigation.yaml). I left a note pointing there so I didn't think we needed to repeat information